### DISCUSSION TITLE
18650 Poor Performance in crazyflie bolt 1.0

### DISCUSSION THREAD

**Discussion by mfxjx000205** [bitcraze/discussions#1043] [Q&A]
Hi, Bitcraze,

In discussion #959, I got a lot of help and built the lab's first large drone according to the guidelines. It's very uplifting.

My large drone configuration is:
**3018 Two-blade propeller**
**1202.5 11000KV motors**
The rest of the configuration is the same as in the [blog](https://www.bitcraze.io/2020/10/testing-crazyflie-bolt-and-1-cell-li-ion/)

We flashed the firmware directly through **cfclient** without changing any parameters such as PID and weight and so on.

During our flight tests, we noticed some strange things:

1. We used the position hold mode for takeoff and limited the maximum thrust to 70%. We find that M1,M3 maintain almost normal thrust. the thrust of M2,M4 always seems to be too much, and in the second half of the flight, the thrust of M2,M4 is always connected to 100%.
2. We found out that fully charged 18650 batteries can only fly steadily for about 14 minutes on the drone in the above configuration, but in the blog, I read that it was 20 minutes. After the flight, I used a multimeter to test the battery voltage. It showed about 3.4V, which is obviously a long way from draining the battery. So I'm speculating that the voltage drop during flight is so large that it can't maintain a stable operating voltage of 3.0V.

We've sourced another 19 bolts from bitcraze, which will be arriving soon and will hopefully enhance the range of the 1s lithium batteries. Anyone have a possible solution to this, thanks a lot!

Regrads!

Hanjie Yu

**Comment by tobbeanton** [comment]
I'll do my best two answer your questions!

1. To me that sounds like it is struggling a bit with stabilizing yaw. Usually this happens if one motor is tilted, contributing to a yaw effect, but this should not happen easily with a setup like yours. Check the mechanics and perhaps propellers.
2. How heavy is the build? Tuning the controller can improve flight time if there is a lot of noise in the motor output. A step-up converter to 5V would be a solution so the battery could go down to e.g. 2.5V but might not be straight forward or even possible to integrate. Two smaller Li-Ion cells in series is not an option?
